src/components/ScriptNav.jsx

```diff
@@ -1,15 +1,15 @@
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
+import { deleteScript } from './api';
 
 const ScriptNav = ({ currentScript, setCurrentScript, scripts, setScripts }) => {
   const [scriptError, setScriptError] = useState('')
 
-  async function deleteScript(event){
+  async function deleteScriptHandler(event){
     const deleteId = event.target.value;
 
     try {
-      const response = await fetch('/api/script/' + deleteId, { method: 'DELETE' });
-
-      if (response.ok) {
+      const response =  await deleteId(deleteId)
+      if (response.ok){
```

Comment:
We should try to have consistent return types from api responses so we don't have to think so much about what each of them are doing. `deleteScript()` returns the raw response object from the fetch while most other functions in `api.js` return json for success or undefined for failure. (Which we should fix by throwing an error on failure or returning some error json for example. Most of the code that uses the functions in `api.js` assumes that it will always succeed and will end up trying to use `undefined` instead of an object.)